src/components/header/header.module.css

```diff
@@ -0,0 +1,136 @@
+.headerWrapper {
+  display: flex;
+  background: #111;
+  width: 100%;
+}
+
+.menuSide {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+}
+
+.burgerButton {
+  display: none;
+}
+
+.authWrapper {
+  display: flex;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+}
+
+.authButton {
+  height: 34px;
+  background: #333;
+  color: var(--white-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 7px;
+  border-radius: 40px;
+}
+
+.authButton:focus {
+  background: var(--orange-color);
+}
+
+.pages {
+  display: flex;
+}
+
+.link {
+  padding: 0;
+}
+
+.activeButton {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.themeWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
+.pipe {
+  height: 20px;
+  border: 1px solid #656565;
+}
+
+.themeButton {
+  padding: 0;
+}
+
+@media screen and (min-width: 200px) and (max-width: 748px) {
```

Comment:
Please check the line-height property, as it might be causing insufficient space between lines on mobile view.
<img width="364" alt="Screenshot 2024-04-27 at 23 01 54" src="https://github.com/alyona-shirpal/fe-react-2024/assets/102523692/fe1cc4b2-4613-4d62-86ec-8403d7141337">